### PR TITLE
Generate BPMGen2 parameters for CFL/WHL

### PR DESCRIPTION
**Pull request by raghavag**
This patch will patch parameters for BpmGen2
based on the example file provided in the BpmGen2
package.

Signed-off-by: Raghava Gudla <raghava.gudla@intel.com>

### CODE REVIEW

File: Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py [hunk 1]
```diff
@@ -97,8 +97,29 @@ def get_config ():
       'ecptr'     :   'EC/ecregionpointer.bin',
       'gbe'       :   'GBE/gbe.bin',
       'bpm'       :   'bpmgen2',
+      'bpm_params':   'bpmgen2/Example.bpDef',
```
Comment (gdong1) [inline Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py]: Where user could get bpmgen2/Example.bpDef?

Comment (raghavag) [reply Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py]: It is from BPMGen2 package. they get it when they download BpmGen2 package

Comment (gdong1) [reply Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py]: Got it.

File: Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py [hunk 2]
```diff
@@ -97,8 +97,29 @@ def get_config ():
...
+      ('AcmRevocation',         '2'),
+      ('NEMPages',              '3'),
+      ('IbbFlags',              '0x2'),
+      ('TxtInclude',            'FALSE'),
+      ('BpmSigScheme',          '0x14:RSASSA'),
+      ('BpmSigPubKey',          'keys\pubkey.pem'),
+      ('BpmSigPrivKey',         'keys\privkey.pem'),
+      ])
+    return params_change_list
+
```
Comment (gdong1) [inline Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py]: Not sure if we could a generic StitchIfwi.py by moving this platform specific config to a config file and let StitchIfwi.py 
 consume it. any idea?

**Pull request merged**